pkg/config/containers.conf

```diff
@@ -216,6 +216,12 @@ default_sysctls = [
 #
 #log_driver = "k8s-file"
 
+# Path to the directory where container log files are stored.
+# If not set, container logs are stored in the default location. 
+# This can be overridden by the --log-opt path=<path> runtime option.
```

Comment:
The text in the man page should match the default config file. Having two different texts seems unnecessary.